<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuE">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste_P"/>
   </widget>
   <widget class="QMenu" name="menuW">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menuH">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuF"/>
   <addaction name="menuE"/>
   <addaction name="menuW"/>
   <addaction name="menuH"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste_P"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/new.png</normaloff>:/myImage/image/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/open.png</normaloff>:/myImage/image/open.png</iconset>
   </property>
   <property name="text">
    <string>OPEN(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/save.png</normaloff>:/myImage/image/save.png</iconset>
   </property>
   <property name="text">
    <string>SAVE(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/saveAs.png</normaloff>:/myImage/image/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>SAVE AS(&amp;A)...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/exit.png</normaloff>:/myImage/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>EXIT(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/undo.png</normaloff>:/myImage/image/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/redo.png</normaloff>:/myImage/image/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/cut.png</normaloff>:/myImage/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/copy.png</normaloff>:/myImage/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/paste.png</normaloff>:/myImage/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close(&amp;O)</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>CloseAll(&amp;A)</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>平铺</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>层叠</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/next.png</normaloff>:/myImage/image/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>下一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/previous.png</normaloff>:/myImage/image/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>前一个</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resource/myImage.qrc">
     <normaloff>:/myImage/image/about.png</normaloff>:/myImage/image/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/myImage.qrc"/>
 </resources>
 <connections/>
</ui>
